src/main/kotlin/no/nav/familie/ba/sak/kjerne/autovedtak/satsendring/SatsendringService.kt

```diff
@@ -24,9 +26,11 @@ class SatsendringService(
             behandlingHentOgPersisterService.hentSisteBehandlingSomErVedtatt(fagsakId)
 
         return sisteVedtatteBehandling == null ||
-            andelerTilkjentYtelseOgEndreteUtbetalingerService
-                .finnAndelerTilkjentYtelseMedEndreteUtbetalinger(sisteVedtatteBehandling.id)
-                .erOppdatertMedSisteSatser()
+            personOpplysningGrunnlagRepository.findByBehandlingAndAktiv(sisteVedtatteBehandling.id)?.let { personOpplysningGrunnlag ->
+                andelerTilkjentYtelseOgEndreteUtbetalingerService
+                    .finnAndelerTilkjentYtelseMedEndreteUtbetalinger(sisteVedtatteBehandling.id)
+                    .erOppdatertMedSisteSatser(personOpplysningGrunnlag)
```

Comment:
Synes det ble litt mye på en gang her. Kunne man brutt det opp litt? Typ:


```suggestion
            return if (sisteVedtatteBehandling == null) {
               true
            } else {        
               val personOpplysningGrunnlagSisteVedtatteBehandling = personOpplysningGrunnlagRepository.findByBehandlingAndAktiv(it.id) ?: error("Fant ikke persongrunnlag på behandling ${it.id}") 
            
              andelerTilkjentYtelseOgEndreteUtbetalingerService
                       .finnAndelerTilkjentYtelseMedEndreteUtbetalinger(sisteVedtatteBehandling.id)
                       .erOppdatertMedSisteSatser(personOpplysningGrunnlagSisteVedtatteBehandling)
            }
```